.pre-commit-config.yaml

```diff
@@ -13,3 +13,8 @@ repos:
         entry: isort
         language: system
         types: [python]
+  - repo: https://github.com/kynan/nbstripout
+    rev: 0.5.0
+    hooks:
+      - id: nbstripout
+        name: Strip notebook output

```

Comment:
As stated, I think I'd rather not strip the Notebook, because it is supposed to be a readable example.